theano/tensor/tests/test_basic.py

```diff
@@ -1696,6 +1698,24 @@ def _numpy_true_div(x, y):
     inplace=True,
     skip=skip_scipy)
 
+ErfcxTester = makeBroadcastTester(
+    op=tensor.erfcx,
+    expected=expected_erfcx,
+    good=_good_broadcast_unary_normal_float_no_complex,
+    grad=_grad_broadcast_unary_normal,
+    eps=2e-10,
+    mode=mode_no_scipy,
+    skip=skip_scipy)
```

Comment:
Here you must update skip to check for scipy version >= 0.12, This is probably what make travis fail.